specs/core/0_beacon-chain.md

```diff
@@ -2176,7 +2158,7 @@ def process_proposer_attestation_rewards(state: BeaconState) -> None:
     for pending_attestations in (state.previous_epoch_attestations, state.current_epoch_attestations):
         for index in get_unslashed_attesting_indices(state, pending_attestations):
             if get_earliest_attestation(state, pending_attestations, index).inclusion_slot == state.slot:
-                base_reward = get_base_reward_from_total_balance(state, get_current_total_balance(state), index)
+                base_reward = get_base_reward(state, get_current_epoch_total_balance(state), index)
```

Comment:
Similar to the other comment above, why are we using the current balance rather than the previous one now?

Reply:
Proposer attestation rewards (which does not fall in the `BASE_REWARDS_PER_EPOCH` because it only affects 64 proposers, and is scaled down by `PROPOSER_REWARD_QUOTIENT`) are handled on a block-by-block basis because the beacon chain can only compute `get_beacon_proposer_index` for the current slot.